velox/exec/tests/utils/AssertQueryBuilder.h

```diff
@@ -183,6 +183,9 @@ class AssertQueryBuilder {
       memory::MemoryPool* pool,
       std::shared_ptr<Task>& task);
 
+  /// Run the query and return the number of result rows.
```

Comment:
Is this to optimize the benchmark execution or? Thanks!

Reply:
Yes, it is used to optimize the benchmark execution. Before that, we only have the API to copy the result, which may interference the benchmark performance.

Reply:
It seems that you use something similar to velox/tool/trace/TraceReplayTaskRunner.h
Maybe rename countResults to runWithoutResults?